snap/snapcraft.yaml

```diff
@@ -1,42 +1,44 @@
 name: shotwell
-version: '0.32.1'
-summary: Open source photo manager for GNOME
-description: |
-  Shotwell is an open source photo manager for GNOME. It allows you to import,
-  organize, edit, and share your photos.
-
+adopt-info: shotwell
 grade: stable
 confinement: strict
-
 base: core22
-adopt-info: shotwell
+
+slots:
+  # for GtkApplication registration
+  drawing:
+    interface: dbus
+    bus: session
+    name: org.gnome.Shotwell
 
 apps:
   shotwell:
-    command: usr/local/bin/shotwell
+    command: usr/bin/shotwell
     extensions: [gnome]
-    desktop: usr/local/share/applications/org.gnome.Shotwell.desktop
+    desktop: usr/share/applications/org.gnome.Shotwell.desktop
     environment:
-      LD_LIBRARY_PATH: "$SNAP/usr/local/lib/x86_64-linux-gnu:\
-        $SNAP/usr/lib/x86_64-linux-gnu"
+      HOME: $SNAP_USER_COMMON
+    plugs:
+      - audio-playback
+      - home
+      - mount-observe
+      - hardware-observe
 
 parts:
   shotwell:
     plugin: meson
     source: https://gitlab.gnome.org/GNOME/shotwell.git
-    source-tag: shotwell-$SNAPCRAFT_PROJECT_VERSION
+    source-tag: shotwell-0.32.1
+    meson-parameters:
+      - --prefix=/usr
+    parse-info: [ usr/share/metainfo/org.gnome.Shotwell.appdata.xml ]
```

Comment:
The linter seems to fail here because of the spaces inside the brackets it looks like